bundles/AdminBundle/Resources/views/Admin/Login/layout.html.php

```diff
@@ -23,8 +23,14 @@
 <body class="pimcore_version_6 <?= $config['branding']['login_screen_invert_colors'] ? 'inverted' : '' ?>">
 
 <?php
-    $backgroundImageUrl = file_exists(PIMCORE_WEB_ROOT . $config['branding']['login_screen_custom_image']) ? $config['branding']['login_screen_custom_image'] : '';
-    if (empty($backgroundImageUrl)) {
+    $customImage = $config['branding']['login_screen_custom_image'];
+    //https://github.com/pimcore/pimcore/issues/8016
+    //https://github.com/pimcore/pimcore/issues/8129
+    if (preg_match('@^https?://@', $customImage) && file_get_contents($customImage)) {
```

Comment:
I woudn't do a `file_get_contents()` here. 
I'd say it's up to the administrator to ensure the image exists.

Reply:
reverted!